math-lib/math/private/functions/lanczos.rkt

```diff
@@ -2,7 +2,7 @@
 
 (require "../../flonum.rkt")
 
-(provide lanczos-sum lanczos-g)
+(provide (all-defined-out))
```

Comment:
I hate to see a specific provide turn into a generic (all-defined-out), unless... does it really make sense here?

Reply:
The module had only 2 definitions, which were provided. I added 2 more, which will also be provided. Instead of writing all 4, I thought to use the generic. If this is not the right way, I see no problem to spelling everything out.

Reply:
We should spell them out. `(all-defined-out)` is a code smell, since it fails to even describe the interface provided.